src/rules/__tests__/unbound-method.test.ts

```diff
@@ -113,33 +113,37 @@ const invalidTestCases: Array<TSESLint.InvalidTestCase<MessageIds, Options>> = [
     ],
   },
   // toThrow matchers call the expected value (which is expected to be a function)
-  ...toThrowMatchers.map(matcher => ({
-    code: dedent`
+  ...toThrowMatchers.map<TSESLint.InvalidTestCase<MessageIds, Options>>(
```

Comment:
We purposely don't define this as it's a complicated type so it's preferred to let typescript infer it

Reply:
Got it. I've addressed it in commit https://github.com/jest-community/eslint-plugin-jest/commit/b030cc748361ecd7692845e22114be041700e9b3.